olive/hardware/accelerator.py

```diff
@@ -57,18 +57,20 @@ class AcceleratorLookup:
         "gpu": [
             "DmlExecutionProvider",
             "CUDAExecutionProvider",
-            "OpenVINOExecutionProvider",
```

Comment:
![image](https://github.com/microsoft/Olive/assets/817030/1f266255-8b7b-460b-8f62-f46b87d59927)

It seems GPU also support OpenVino? why do we need remove it?